examples/helia-nextjs/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "helia": "^4.0.1",
-    "next": "^14.1.0",
+    "next": "^14.0.4",
```

Comment:
This is still the same semver range - you need to remove the caret to pin it to a version.